src/System.Management.Automation/utils/StructuredTraceSource.cs

```diff
@@ -1074,20 +1223,16 @@ internal void WriteLine(object arg)
 
                 if (format != null)
                 {
-                    output.AppendFormat(
-                        CultureInfo.CurrentCulture,
-                        format,
-                        args);
+                    output.AppendFormat(CultureInfo.CurrentCulture, format, arg1, arg2, arg3, arg4, arg5, arg6);
```

Comment:
`StringBuilder` doesn't even only have `arg0`, `arg0, arg1`, and `arg0, arg1, arg2` 3 overloads. Why are you doing the 6 args here?